moveit_ros/moveit_servo/include/moveit_servo/servo.hpp

```diff
@@ -134,8 +134,11 @@ class Servo
 private:
   /**
    * \brief Convert a give twist command to planning frame
-   * (This implementation assumes that source and target frames are stationary)
+   * This implementation does not differentiate between body twist and spatial twist,
+   * i.e if the command is as seen from a stationary or non-stationary frame.
+   * The result of this transformation will only be accruate if the command frame is a stationary frame.
    * See issue: https://github.com/ros-planning/moveit2/issues/2150
+   * TODO: Update with proper transformations.
```

Comment:
I'd suggest deleting the TODO b/c we know you're going to be fixing it in the next few days, and the fix isn't going to be in this file anyway.

```suggestion
```